### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by phadej**
I noticed in #161 that travis isn't reliable, let's see if this turns it into green.

### CODE REVIEW

**Comment by phadej** [comment]
Green!

**Comment by phadej** [comment]
Cannot test GHC-7.6.3 and GHC-7.4.2 as that requires re-installing `bytestring` which isn't possible due `doctest` (which depends on `ghc`).

**Comment by snoyberg** [comment]
This is Kazu's repo, but I'm opposed in general to the idea of only testing with new-build.

File: fast-logger/fast-logger.cabal [hunk 1]
```diff
@@ -48,6 +49,7 @@ Test-Suite spec
 
     Ghc-Options:     -Wall -threaded
     Other-Modules:   FastLoggerSpec
+    build-tool-depends: hspec-discover:hspec-discover
```
Comment (kazu-yamamoto) [inline fast-logger/fast-logger.cabal]: This field breaks the backword compatibility of old Cabals.
Old cabal-install abnormally exits if this field exists.

**Comment by kazu-yamamoto** [comment]
Since the `network` package already uses `new-build` only, I' not nervous  about this PR.
But I would like to ask motivation of this PR to @phadej.
Is this the way to go for Cabal?

**Comment by phadej** [comment]
Sorry for not responding promptly.

The motivation is that current travis setup is not working, and the PR uses https://github.com/haskell-ci/haskell-ci which is actively maintained travis CI script generator. Previous version used HVR PPA, so does this one. `cabal new-build` is _the_ way for `cabal`.

**Comment by phadej** [comment]
Kazu: How build-tool-depends:hspec-discover bug manifests itself?

**Comment by kazu-yamamoto** [comment]
`cabal install hspec-discover` instead of `build-tool-depends: hspec-discover`.
See the master branch.

**Pull request merged**